### PR TITLE
implemented as_table(light=True)

### PR DESCRIPTION
**Pull request by gdementen**
this will be useful to implement optional lighter display

if we do not want to make this the default, I ma unsure about the final user-facing API, but this can come later

### CODE REVIEW

File: larray/core.py [hunk 1]
```diff
@@ -5965,6 +5964,22 @@ def as_table(self, maxlines=None, edgeitems=5):
...
+         ['a0', 'b0', 0, 1, 2],
+         ['a0', 'b1', 3, 4, 5],
+         ['a1', 'b0', 6, 7, 8],
+         ['a1', 'b1', 9, 10, 11]]
+        >>> list(arr.as_table(light=True))  # doctest: +NORMALIZE_WHITESPACE
+        [['a', 'b\\\\c', 'c0', 'c1', 'c2'],
+         ['a0', 'b0', 0, 1, 2],
+         ['', 'b1', 3, 4, 5],
+         ['a1', 'b0', 6, 7, 8],
+         ['', 'b1', 9, 10, 11]]
```
Comment (alixdamman) [inline larray/core.py]: Why using list in `as_table` examples?

File: larray/utils.py [hunk 2]
```diff
@@ -212,6 +212,35 @@ def rproduct(*i):
     return product(*[x[::-1] for x in i])
 
 
+def light_product(*iterables, repeat=1):
+    """Cartesian product of input iterables, replacing repeated values by empty strings.
```
Comment (alixdamman) [inline larray/utils.py]: signature not accepted in Python 2.7

Comment (gdementen) [inline larray/utils.py]: yes... Will fix this...

File: larray/core.py (hunk 1 above)
Comment (gdementen) [inline larray/core.py]: because it currently return a generator

**Comment by gdementen** [comment]
having something like set_printoptions was one of the ideas. We could also have our own function to display arrays to the console, instead of print(), so that we can pass it arguments. See other issue about this.

**Pull request merged**